File: kot/src/test/java/org/dan/jadalnia/sys/type/number/ImmutableNumberTest.java
package org.dan.jadalnia.sys.type.number;

import org.hamcrest.core.Is;
import org.junit.Test;

import static org.junit.Assert.assertThat;

public class ImmutableNumberTest {
    @Test
    public void equals() {
        assertThat(new ImmutableNumber(1),
                Is.is(new ImmutableNumber(1)));
    }
}

File: kot/src/test/kotlin/org/dan/jadalnia/sys/type/number/ImmutableNumberTest.kt
package org.dan.jadalnia.sys.type.number

import assertk.assertThat
import assertk.assertions.isEqualTo
import org.junit.Test

class ImmutableNumberTest {
  @Test
  fun equals() = assertThat(ImmutableNumber(1)).isEqualTo(ImmutableNumber(1))
}
